nova/static/memberJSON/aa753.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please do not commit these generated json

Reply:
Thank you for catching this! I've removed json files and added the same to `.gitignore`